l5kit/l5kit/tests/geometry/transform_test.py

```diff
@@ -2,34 +2,43 @@
 import pytest
 import transforms3d
 
-from l5kit.geometry import transform_point, transform_points, transform_points_transposed, world_to_image_pixels_matrix
+from l5kit.geometry import transform_point, transform_points, transform_points_transposed
+from l5kit.rasterization.render_context import RenderContext
 
 
 def test_transform_to_image_space_2d() -> None:
```

Comment:
I would move this unit test to `l5kit/tests/rasterization/render_context_test.py`